sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -10,7 +10,24 @@
 #include <sound/soc-acpi-intel-match.h>
 #include "soc-acpi-intel-sdw-mockup-match.h"
 
+static const struct snd_soc_acpi_codecs mtl_max98357a_amp = {
+	.num_codecs = 1,
+	.codecs = {"MX98357A"}
+};
+
+static const struct snd_soc_acpi_codecs mtl_rt5682_rt5682s_aic = {
```

Comment:
How about calling it "mtl_rt5682_rt5682s_hp"? Just like other platforms.

Reply:
> How about calling it "mtl_rt5682_rt5682s_hp"? Just like other platforms.

Ack